### PR TITLE
Remove deprecated utility

### PR DESCRIPTION
**Pull request by spiess-demos**
**Ticket:** https://yaits.demos-deutschland.de/T30974

This PR removes deprecated utility classes for border-radius.

- `border-radius--6` did not exist anyhow, it is replaced with the largest border-radius demosplan-ui provides
- `border-radius-extra-large` was meant to create a fully rounded pill-like element, demosplan-ui/Tailwind provides `rounded-full` for this purpose
- without installed addons, there was an error in SegmentsList, this is catched with a condition now

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

File: client/js/components/procedure/SegmentsBulkEdit/SelectedTagsList.vue [hunk 1]
```diff
@@ -10,8 +10,8 @@
 <template>
   <ul class="o-list">
     <li
-      class="inline-block border bg-color--white color--grey-dark"
-      style="border-radius: 4px; padding: 4px 8px; margin-right: 12px;"
+      class="inline-block border bg-color--white color--grey-dark rounded-md"
+      style="padding: 4px 8px; margin-right: 12px;"
```
Comment (gruenbergerdemos) [inline client/js/components/procedure/SegmentsBulkEdit/SelectedTagsList.vue:14]: Could we remove the style here completely?

**Review by gruenbergerdemos — COMMENTED** [review]
Just one question, otherwise lgtm

File: client/js/components/procedure/SegmentsBulkEdit/SelectedTagsList.vue (hunk 1 above)
Comment (spiess-demos) [reply client/js/components/procedure/SegmentsBulkEdit/SelectedTagsList.vue]: sure! 5873417dae98012b0e20893dc53e67975bb1a69e

**Review by gruenbergerdemos — APPROVED** [review]
lgtm

**Pull request merged**